mermithid/processors/TritiumSpectrum/FakeDataGenerator.py

```diff
@@ -315,9 +383,7 @@ def generate_unbinned_data(self, Q_mean, mass, ROIbound, S, B_1kev, nsteps=10**4
         time0 = time.time()
 
         if array_method == True:
-            ratesS = convolved_spectral_rate_arrays(self.Koptions, Q_mean, mass, Kmin,
-                                                    lineshape, params, min_energy, max_energy,
-                                                    self.complexLineShape, self.final_state_array)
+            ratesS = convolved_spectral_rate_arrays(self.Koptions, Q_mean, mass, Kmin, lineshape, params, min_energy, max_energy, self.complexLineShape, self.final_state_array)
```

Comment:
personally I prefer linebreaks to keep lines short. but there is nothing wrong with leaving it all in one line.

Reply:
Makes sense – I put back the linebreaks (and added some to the similar lines below, for consistency).